src/features/workflow/WorkflowOutline.tsx

```diff
@@ -24,59 +25,61 @@ const WorkflowOutline: React.FC<WorkflowOutlineProps> = ({
+              </div>
+            </ScrollTrigger>
           </div>
-        </ScrollTrigger>
-      </div>
-      <div className='mb-2'>
-        {workflow && workflow.steps?.map((step: TransformStep, i: number) => {
-          let r
-          if (run) {
-            r = (run?.steps && run?.steps.length >= i && run.steps[i]) ? run.steps[i] : NewRunStep({ status: "waiting" })
-          }
-          return (
-            <ScrollTrigger target={step.name} key={i}>
+          <div className='mb-2'>
+            {workflow && workflow.steps?.map((step: TransformStep, i: number) => {
+              let r
+              if (run) {
+                r = (run?.steps && run?.steps.length >= i && run.steps[i]) ? run.steps[i] : NewRunStep({ status: "waiting" })
+              }
+              return (
+                <ScrollTrigger target={step.name} key={i}>
+                  <div className='text-sm mb-0.5 text-qrigray-400 capitalize'>
+                    {step.name}
+                    {r && <div className='float-right text-green-500'><RunStatusIcon state={r.status || "waiting"} /></div>}
+                  </div>
+                </ScrollTrigger>
+              )
+            })}
+            {runMode === 'save' && <ScrollTrigger target='save'>
               <div className='text-sm ml-2 mb-1 text-gray-500 font-semibold'>
-                <span className='font-black text-gray-500'>{i+1}</span> &nbsp; {step.name}
-                {r && <div className='float-right text-green-500'><RunStatusIcon state={r.status || "waiting"} /></div>}
+                <span className='font-black text-gray-500'>{((workflow && workflow.steps?.length) || 0)+1}</span> &nbsp; save
+                {/* {r && <div className='float-right text-green-500'><RunStateIcon state={r.status || RunState.waiting} /></div>} */}
               </div>
-            </ScrollTrigger>
-          )
-        })}
-        {runMode === 'save' && <ScrollTrigger target='save'>
-          <div className='text-sm ml-2 mb-1 text-gray-500 font-semibold'>
-            <span className='font-black text-gray-500'>{((workflow && workflow.steps?.length) || 0)+1}</span> &nbsp; save
-            {/* {r && <div className='float-right text-green-500'><RunStateIcon state={r.status || RunState.waiting} /></div>} */}
+            </ScrollTrigger>}
           </div>
-        </ScrollTrigger>}
-      </div>
-      <ScrollTrigger target='on-completion'><div className='font-semibold text-gray-900 mb-2 uppercase text-xs tracking-wide'>On Completion</div></ScrollTrigger>
-      <div className='text-xs text-gray-500 bg-gray-200 inline-block py-0 px-2 rounded-xl border ml-2'>
-        <Icon icon='cloudUpload' size='sm' className='mr-1'/> <span>push to cloud</span>
+          <ScrollTrigger target='on-completion'><div className='font-semibold text-qrinavy mb-2'>On Completion</div></ScrollTrigger>
+
+          <div className="mb-20">
+            {workflow && <DeployButtonWithStatusDescription workflow={workflow} />}
```

Comment:
Still not a fan of having the deploy button on the sidebar, but let's take that debate up when chatting with the rest of product squad. Fine to leave this in for now